routers/web/web.go

```diff
@@ -236,6 +236,10 @@ func RegisterRoutes(m *web.Route) {
 	if setting.Federation.Enabled {
 		m.Get("/.well-known/nodeinfo", NodeInfoLinks)
 	}
+	m.Get("/.well-known/change-password", func(w http.ResponseWriter, req *http.Request) {
+		http.Redirect(w, req, "/user/settings/account", 307)
```

Comment:
Seems like the proposal allows a 307, but that's a `Temporary Redirect` status code, which I find a bit weird for a "permanent redirect" in this case, as we can almost guarantee that this URL will be a redirect and not actually a rendered page. 302 or 303 seems to make a bit more sense in this case.

Reply:
I don't understand all the minute details about 3xx status codes but I personally prefer to use 307/308 because they don't introduce method changes. Thought in this case, method should probably always be GET, so 302 may just be fine.

Reply:
> I don't understand all the minute details about 3xx status codes

They all boil down to the same conclusion, get the f*ck out of here, and please go to: xxx via the method that is linked to the statusCode.

Reply:
I don't believe the spec allows 308, so I'm on same page as @silverwind at keeping it 307. Although I'd recommend using the constant from the http package (eg. http.StatusTemporaryRedirect or something like that instead of hard coding a number)

Reply:
It shouldn't be `Temporary Redirect` here in my mind. What if we change the target location between releases.

Reply:
Sorry I typo-ed before, I mean `Temporary Redirect` should be correct  ... 😭

Reply:
Sure thing then

Reply:
I switch to a constant.